app/Http/Controllers/Users/UserFilesController.php

```diff
@@ -139,18 +139,25 @@ public function show($userId = null, $fileId = null)
 
         // the license is valid
         if (isset($user->id)) {
+
             $this->authorize('view', $user);
 
             $log = Actionlog::find($fileId);
-            $file = $log->get_src('users');
+            $file = 'private_uploads/users/'.$log->filename;
```

Comment:
By using the `get_src`, it prepends `/storage`, which breaks the inline. We don't have to use the `get_src()` anywhere else, so I'm not sure why it donks here. (We also have to account for S3 and cloud storage paths, so it's not as straightforward as it might seem)

Reply:
Yeah I'm not sure what is going on...dropping `return StorageHelper::downloader($file);` in place of `return Response::download($file);` on line 156 gets downloads working locally for me again but, like we talked about, there is other considerations to take before doing the swap.